src/typings-new-format/spec-1.0/datamodel.ts

```diff
@@ -122,20 +127,26 @@ export interface TypeDeclaration extends common.Annotable {
      * If displayName is not specified, it defaults to the element's key
      * (the name of the property itself
      */
-    displayName?: string
+    displayName: string
```

Comment:
It has been agreed that TypeDeclarations must always provide dislpayName value even if it has not been specified by the user -- in this case it just equils name value. Other RAML components are not required to do so.